### ISSUE TITLE
Add support for disabled IPv6 network stack

### ISSUE THREAD

**Issue by SisyphusIsntHappy** [safing/portmaster#437]
I installed Portmaster on Archlinux using the PKGBUILD provided by the developers. 
I have disabled IPV6 in linux kernel. Portmaster throws me this error,
```
can't initialize ip6tables table `mangle': Address family not supported by protocol
Perhaps ip6tables or your kernel needs to be upgraded.
```
When I enable IPV6, Portmaster works fine, but I don't want IPV6

**How did you reproduce it?**:
Run Portmaster with IPV4 only, by disabling IPV6

Platform :
Archlinux, Kernel - 5.10.75-1-lts

How do I use Portmaster with IPV4 only?

**Comment by dhaavi** [comment]
Hey @SisyphusIsntHappy, thanks for raising this issue.

This is an interesting and very good use case. I thought about it and am not sure what the best way is to handle this.
The problem is, that when we just disable IPv6 integration when IPv6 is not available, we will not be able to protect the user when he decides to enable IPv6 while the Portmaster is running - as the integration would not be set up.
Polling for network stack changes and enabling/disabling IPv6 integration on the fly introduces a lot complexity and is a can of worms.
We will need some time to figure this out.

I will convert this to a suggestion, as this is currently expected behavior.

**Workaround**
As a workaround you can enable IPv6 and just add a "Block `::/0`" rule to both  [Outgoing Rules](https://docs.safing.io/portmaster/settings#filter/endpoints) and [Incoming Rules](https://docs.safing.io/portmaster/settings#filter/serviceEndpoints) as the first rule. This will block all IPv6 traffic.

**Comment by lreading** [comment]
Understanding that IPv6 disabled linux users are probably not your biggest user group, I think it would be nice if there were a more user-friendly error message/notification. :smile:  I suspect this will happen on most machines that are following a CIS benchmark. I just ran into this when trying to install it on an Ubuntu 20.04 machine with ipv6 disabled.  

This is my first time trying out Portmaster, and it took me longer than I'd like to admit to figure out what was actually going on.  I was hoping to find a way of forcing it to work, until I found this issue.  It wasn't immediately obvious to me that disabling IPv6 would break the application; a more specific message (maybe including the workaround posted above) would save those few users some time.

Now that I have it running, it's fantastic and I absolutely love it!  Keep up the awesome work!
